src/components/Swap/Summary/index.tsx

```diff
@@ -146,7 +147,7 @@ function ConfirmButton({
         },
       }}
     >
-      <Trans>Confirm swap</Trans>
+      {isPending ? <Trans>Confirm</Trans> : <Trans>Confirm swap</Trans>}
```

Comment:
this seems like an odd copy distinction to me 🙃

Reply:
this seems weird to me too... can we get design to check this